lib/map-file-delphi.ts

```diff
@@ -22,7 +22,7 @@
 // ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 // POSSIBILITY OF SUCH DAMAGE.
 
-import { MapFileReader } from './map-file';
+import {MapFileReader} from './map-file';
```

Comment:
I assume this is some automatic intellij refactoring? Can you disable it?

Reply:
Got it: had to copy some settings I had for JS to TS.